### ISSUE TITLE
Citation Key values always even numbers

### ISSUE THREAD

**Issue by Archilegt** [NaturalHistoryMuseum/scratchpads2#6529]
**Describe the bug**
As per the title. Citation Key values are being added on a "plus two" fashion, which in my case is creating series of even numbers. The three latest references have gotten citekeys 1444, 1446, and 1448. See:
1444: https://myriatrix.myspecies.info/content/description-dune-nouvelle-esp%C3%A8ce-diule
1446: https://myriatrix.myspecies.info/content/redescription-rhysida-celeris-humbert-saussure-1870-proposal-eight-new-synonyms
1448: https://myriatrix.myspecies.info/content/myriapoda-nova-americana-series-2

Could this be fixed? It gives the wrong idea regarding how many references are in the system and how fast the reference list is growing.

**Comment by therobyouknow** [comment]
Yes @Archilegt I agree that looks like a bug.  I think it will be possible to fix.  
At the moment, I can't think of any valid reason why it would increment by 2.

**Comment by Archilegt** [comment]
Maybe related to https://github.com/rdmpage/biostor/issues/100#issuecomment-1263413865
> @Archilegt No, I think the problem is with Drupal. If I go to any ScratchPad, click on the "LITERATURE" tab and then click on "Export selection: on the left (to get the whole bibliography) I get duplicates. There is something deeply broken here.

It seems that the Drupal biblio module in Scratchpads has been creating duplicated references in the background and those are getting exported. We need to identify the root cause and run a literature deduplication. The complication is that if the CiteKeys were already used to write text, e.g., in Taxon Descriptions and we wipe the wrong duplicate and corresponding CiteKey, that will break the citation in text and the automatic creation of a reference list.

**Comment by Archilegt** [comment]
To reproduce: Please find a Scratchpad with a small literature database, so that we can easily check the export file visually.

**Comment by Archilegt** [comment]
I run a duplication analysis with the Myriatrix RIS export file and Zotero. Documentation is at https://github.com/rdmpage/biostor/issues/100#issuecomment-1263458090
This is simply horrible. References duplicated 26 times, and after deduplication only the tiny amount of 50 Myriatrix references remain. Where are the 1309 references and why are they not getting exported?
Le ciel lui tombe sur la tête!
